dev-util/hip/files/hip-5.7.0-fix-unaligned-memcpy.patch

```diff
@@ -0,0 +1,56 @@
+--- a/rocclr/device/rocm/rocvirtual.cpp
```

Comment:
Please report this upstream and include a link in the patch.

The patch should also have an explanation contained within it.

Reply:
Added to patch; link for backlink: https://github.com/ROCm-Developer-Tools/clr/issues/18